### PR TITLE
Add `quick` arg to sacrifice optimal behaviors for quicker execution

### PR DESCRIPTION
**Pull request by Rinn**
Only change currently is to instead use historical price rather than getSalePrice if the prices are less than a week old, for only items that will be sold.

### CODE REVIEW

**Review by horrible-little-slime — APPROVED** [review]
Is this tested? It looks generally good

**Comment by Rinn** [comment]
I'll test this after rollover, i'm in the middle of a run atm

**Comment by Rinn** [comment]
I'll follow up after it finishes running, but it did blitz through all the dailies super fast, most notably not searching for anything when it open cargo shorts. The only thing I think may cause issue is the 50k maximize limit and the longer maximize query because of all the -equip brimstone, but i'll know more later.

**Comment by horrible-little-slime** [comment]
Can/should we closet brimstone instead? I guess if we do that we should do
papier mache as well

**Comment by Rinn** [comment]
We don't currently disable acquiring from closet so that wouldn't make a difference. My wrapper script put them in my display but that's really dumb, also if I remember correctly i think fixed the issue with long maximize strings in mafia a while ago.

**Comment by Rinn** [comment]
I had some issues due to a power outage so I'll test this again after another run in a few days.

**Comment by docrostov** [comment]
@phulin gurbo is lit

File: src/session.ts [hunk 1]
```diff
@@ -147,14 +155,24 @@ function printSession(session: Session): void {
   printProfit(highValue);
   print(` You lost meat on ${lowValue.length} items including:`);
   printProfit(lowValue);
+  print(`Quick mode was enabled, results may be less accurate than normal.`);
+}
+
+function garboSaleValue(item: Item) {
+  if (globalOptions.quickMode) {
+    if (historicalAge(item) <= 7.0 && historicalPrice(item) > 0) {
+      return historicalPrice(item) * 0.9;
```
Comment (horrible-little-slime) [inline src/session.ts:164]:
```suggestion
      const isMallMin = historicalPrice(item) === Math.min(100, 2 * autosellPrice(item));
      return isMallMin ? autosellPrice(item) : 0.9 * historicalPrice(item);
```

**Review by horrible-little-slime — CHANGES_REQUESTED** [review]
One tiny change

**Review by horrible-little-slime — APPROVED** [review]
yell heah

**Pull request merged**